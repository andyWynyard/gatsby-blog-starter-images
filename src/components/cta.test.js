import React from "react";
import renderer from "react-test-renderer";
import CTA from "./cta";

describe("CTA", () => {
  it("Renders Correctly", () => {
    const data = {
      site: {
        siteMetadata: {
          peterContact:
            "mailto:peter.knapp@sigma.se?Subject=Jag%20vill%20h√∂ra%20mer%20om%20UX"
        }
      }
    };

    const { peterContact } = data.site.siteMetadata;

    const tree = renderer.create(<CTA contact={peterContact} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
