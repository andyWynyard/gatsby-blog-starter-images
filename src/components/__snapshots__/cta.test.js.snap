// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CTA Renders Correctly 1`] = `
<div
  className="cta
      "
>
  <div
    className="
          cta__content
        "
  >
    <h6
      className="cta__statement"
    >
      Your Creative Digital Experts
    </h6>
    <h3
      className="cta__question"
    >
      Find a consultant you found interesting?
    </h3>
    <a
      className="css-uxi9oy"
      color="white"
      href="mailto:peter.knapp@sigma.se?Subject=Jag%20vill%20hÃ¶ra%20mer%20om%20UX"
    >
      Lets Talk
    </a>
  </div>
</div>
`;
